tensorflow_recommenders_addons/dynamic_embedding/core/kernels/redis_impl/redis_cluster_connection_pool.hpp

```diff
@@ -172,8 +172,6 @@ class RedisWrapper<RedisInstance, K, V,
       } catch (const std::exception &err) {
         LOG(ERROR) << "RedisHandler error in PipeExecRead for slices "
                    << hkey.data() << " -- " << err.what();
-        error_ptr = std::current_exception();
```

Comment:
These are some bugs that already fixed in the main branch. This file needs to be merged commit from the main branch.